README.md

```diff
@@ -193,7 +193,8 @@ This will delete all resources in the created namespaces and if the kubernetes c
 Please note that cleaning up the resources will take some time.
 
 ## CONTRIBUTION
- TBD
+
+Find the contributing guidelines here[https://github.com/openMF/mojafos/blob/master/CONTRIBUTING.md]
```

Comment:
Kindly fix this link. It does not render well

Reply:
Once this PR is merged, everything will function as intended.

Reply:
Similar to how README.md contains the route https://github.com/openMF/mojafos/blob/master/README.md, CONTRIBUTING.md will have a similar link with the file name changed at the end.